docs/visualization/representations-and-tables.ipynb

```diff
@@ -252,23 +252,154 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "In addition, Variables have a compact string format:"
+    "The format of variables can be controlled using f-strings or [format](https://docs.python.org/3/library/functions.html?highlight=format#format).\n",
+    "For example, the default format shows the first 2 and last 2 elements:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "var = sc.linspace('x', 0.0, 1.0, 11, unit='m')\n",
+    "f'{var}'"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Use `<` to show the first 4 elements:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "f'{var:<}'"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Use `#n` to show `n` elements instead of 4:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "f'{var:#5}'"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Configure how elements are formatted: (Note the double colon!)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "f'{var::.1e}'"
```

Comment:
Add a note that the bits after the second `:` is just the normal spec that you can use in Python (and add a link?)